spec/requests/user_badges_controller_spec.rb

```diff
@@ -323,16 +329,22 @@
 
       put "/user_badges/#{user_badge.id}/toggle_favorite.json"
       expect(response.status).to eq(200)
+      parsed = response.parsed_body
+      expect(parsed["user_badge"]["is_favorite"]).to be false
       expect(user_badge.reload.is_favorite).to eq(false)
       expect(user_badge2.reload.is_favorite).to eq(false)
 
       put "/user_badges/#{user_badge.id}/toggle_favorite.json"
       expect(response.status).to eq(200)
+      parsed = response.parsed_body
+      expect(parsed["user_badge"]["is_favorite"]).to be true
       expect(user_badge.reload.is_favorite).to eq(true)
       expect(user_badge2.reload.is_favorite).to eq(true)
 
       put "/user_badges/#{other_user_badge.id}/toggle_favorite.json"
       expect(response.status).to eq(200)
+      parsed = response.parsed_body
+      expect(parsed["user_badge"]["is_favorite"]).to be true
```

Comment:
```suggestion
      expect(parsed["user_badge"]["is_favorite"]).to eq(true)
```

Let's follow this syntax since the surround code is already doing so. Feels weird to have 2 different syntax within the same file.

Reply:
It's a fair point 👍